### ISSUE TITLE
I want to use the SDXL model, how can I modify code?

### ISSUE THREAD

**Issue by coollofty** [magic-quill/MagicQuill#89]
The overall drawing effect of SD1.5 is limited, while SDXL is better. How can I switch to SDXL? I see a brush net

**Comment by SesioN** [comment]
Yeah it would be awesome to have SDXL, SD3.5 or Flux working with this tool.

**Comment by Pythonpa** [comment]
Hope to be realized

**Comment by clevnumb** [comment]
Especially FLUX...please.

**Comment by kenantang** [comment]
Here's a tool we develop to support SDXL and FLUX dev https://github.com/kenantang/spice
